### ISSUE TITLE
Nested list is rendering as code

### ISSUE THREAD

**Issue by nicotsou** [markedjs/marked#1503]
**Describe the bug**
I noticed that sometimes the nested lists are rendered as code. 

**To Reproduce**

By pasting the following lines in the online editor:

```

 *  Some very important:  
      
    
    
     *  One;
     *  two;
     *  three
```

The following result will be displayed:
![image](https://user-images.githubusercontent.com/1083396/60030689-b6125080-96a3-11e9-93e9-9a3794ee7d7d.png)

**Expected behavior**
We should not display nested list items as code:

![image](https://user-images.githubusercontent.com/1083396/60030747-d3dfb580-96a3-11e9-9356-83737c2878a7.png)

This is the behavior we get in VS Code:

![image](https://user-images.githubusercontent.com/1083396/60030886-0ee1e900-96a4-11e9-81ef-b1da3ed8562c.png)

**Comment by UziTech** [comment]
marked sees the multiple blank lines as the end of a list.

[commonmark demo](https://spec.commonmark.org/dingus/?text=%20*%20%20Some%20very%20important%3A%0A%0A%0A%20%20%20%20%20*%20%20One%3B%0A%20%20%20%20%20*%20%20two%3B%0A%20%20%20%20%20*%20%20three)
[marked demo](https://marked.js.org/demo/?text=%20*%20%20Some%20very%20important%3A%0A%0A%0A%20%20%20%20%20*%20%20One%3B%0A%20%20%20%20%20*%20%20two%3B%0A%20%20%20%20%20*%20%20three&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=master)

**Comment by nicotsou** [comment]
Why the exact example is being displayed as nested lists in many other markdown editors? Is there any spec that defines this behavior?

**Comment by calculuschild** [comment]
This should be fixed by #2112
